README.md

```diff
@@ -138,30 +146,6 @@ There should be exactly one blank line between methods to aid in visual clarity
 
 If required, `@synthesize` and `@dynamic` should each be declared on new lines in the implementation.
 
-Colon-aligning method invocation should often be avoided.  There are cases where a method signature may have >= 3 colons and colon-aligning makes the code more readable. Please do **NOT** however colon align methods containing blocks because Xcode's indenting makes it illegible.
```

Comment:
The obsessive compulsive in me still has trouble with indented blocks no matter how short, but if there's no way to enforce with the linter I might just have to tell my OCD to shut up.